my-school/src/Components/ActivityCard.js

```diff
@@ -11,44 +14,192 @@ import {
     ModalBody,
     ModalCloseButton,
     SlideIn,
-    Tag
+    Tag,
+    FormControl,
+    FormLabel,
+    Textarea,
+    NumberInput,
+    NumberInputField,
+    NumberInputStepper,
+    NumberIncrementStepper,
+    NumberDecrementStepper,
+    Box,
+    Flex
 } from '@chakra-ui/core';
-import axios from 'axios';
-
-export default function ActivityCard(props)  {
+import '../App.css';
 
+function ActivityCard(props)  {
 
+    const [input, setInput]= useState();
```

Comment:
```suggestion
    const [input, setInput]= useState();
    const [hour, setHour] = useState();
    const [min, setMin] = useState();
    const [month, setMonth] = useState();
    const [day, setDay]= useState();
    const [year, setYear] = useState();
    const [ selectedFile , setSelectedFile]= useState(null)
```
These state values should be initialized to values of the proper data type. It might not be a problem for all of them, but at the very least we'll see a red react error from controlled inputs related to the `input` state.

Reply:
i just changed it, thanks mike